src/main/java/org/vanilladb/core/query/planner/Planner.java

```diff
@@ -31,78 +31,85 @@
 

```

Comment:
There may be a way to implement the logic of ShowPlanner by using QueryPlanner only.